packages/ERTP/test/unitTests/test-mintObj.js

```diff
@@ -1,9 +1,12 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-
 import '@agoric/install-ses';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { test } from 'tape-promise/tape';
+import { makeIssuerKit } from '../../src/issuer';
+import { MathKind } from '../../src/amountMath';
 
-import makeIssuerKit from '../../src/issuer';
+const { STR_SET, SET } = MathKind;
```

Comment:
Given how little these constants are used, i suspect it would be clarifying to call them "MATH_SET" and the like.

Reply:
Actually, this is the only place I used them without the `MathKind` prefix. When used that way, `MATH_` isn't helpful. I'll just add the `MathKind` prefix here.